### ISSUE TITLE
Fatal Error in WebStorm

### ISSUE THREAD

**Issue by RavenHursT** [hillelcoren/redux-intellij-plugin#1]
Installed the plugin. Immediately got this error report:

![image](https://user-images.githubusercontent.com/496983/53359723-62604800-38f9-11e9-8a08-48044c8ebb09.png)

**Comment by hillelcoren** [comment]
Thanks for reporting it, I'm not sure the cause.

Have you created any projects yet?

**Comment by RavenHursT** [comment]
Yes.. I think it may have something to do w/ the plugin assuming that all
projects will have a /lib folder?  Not all Redux projects are
configured/structured that way.
